### PR TITLE
[AAE-3831] Select last version for process definitions

### PR DESCRIPTION
**Pull request by pmartinezga**

### CODE REVIEW

File: activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java [hunk 1]
```diff
@@ -824,6 +822,20 @@ public void should_handleBigDecimalAndDoubleAndLocalDateTimeVariables() {
                 tuple("doubleVar", doubleValue, "double"),
                 tuple("localDateTimeVar", localDateTime, "localDateTime"),
                 tuple("localDateVar", localDate, "localDate")
-                );
+            );
+    }
+
+    @Test
+    public void should_processAdminOnlyProcessDefinitionsFromLatestVersionRetrieved() {
```
Comment (erdemedeiros) [inline activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java:829]: In addition to this test, I think you need one similar to [this one](https://github.com/Activiti/Activiti/blob/develop/activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/ApplicationUpgradeIT.java#L101-L118) covering the method `getProcessDefinition` from admin API.

**Review by erdemedeiros — CHANGES_REQUESTED** [review]

File: activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java (hunk 1 above)
Comment (pmartinezga) [reply activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java:829]: done, thanks for the suggestion!

**Review by erdemedeiros — APPROVED** [review]
LTGM!

**Comment by erdemedeiros** [comment]
Ref https://github.com/Activiti/Activiti/issues/3594.

**Pull request merged**